spec/controllers/api/v1/organisations_user_controller_spec.rb

```diff
@@ -65,5 +68,24 @@
       expect(response.status).to eq(422)
       expect(subject["errors"]).to eq("User email is invalid")
     end
+
+    it "add organisation user with existing mobile from User Model" do
```

Comment:
We should replace `add` word with `adds`

Reply:
Also, I think we need to check other data as well like what happens in case of first_name, last_name. etc.  Let me know your thoughts.

Reply:
You mean different test case without first_name and last_name ?